charts/fluid/fluid/values.yaml

```diff
@@ -9,6 +9,9 @@ crdUpgrade:
 image:
   imagePullSecrets: []
 
+installation:
```

Comment:
helm charts values are all installation dependent, 
so direct use of namespace is recommended

Reply:
done